client/reader/start/controller.js

```diff
@@ -30,8 +30,7 @@ export function start( context ) {
 		ensureStoreLoading( feedStore, context );
 	}
 
-	setPageTitle( i18n.translate( 'Start' ) );
-
+	setPageTitle( context, i18n.translate( 'Start' ) );
```

Comment:
`setPageTitle` is passed as a prop in the component below and should probably be removed.